src/components/form/bounty/index.tsx

```diff
@@ -231,6 +231,7 @@ function Form(props: FormProps) {
                     isFocused={isFocused}
                     errors={errors}
                     scrollToTop={scrollToTop}
+                    github_state={values.github_description ? true : false}
```

Comment:
Great work @VividhPandey003, Make this logic `values.github_description ? true : false` a variable and reuse it when it is needed, you don't have to repeat yourself, other than this, it works perfectly.

Reply:
Okay, I'll do the needful. Thanks for the review.😄

Reply:
Hey @elraphty I've made the required changes.